src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingPreview.jsx

```diff
@@ -102,6 +109,14 @@ const EmbeddingPreview = (props) => {
     }
   }, [selectedPlot]);
 
+  useEffect(() => {
+    Object.keys(plots).forEach((key) => {
+      if (key.notSaved) {
+        dispatch(savePlotConfig(experimentId, key.plotUuid));
+      }
```

Comment:
this needs to be throttled, or otherwise we will be sending too many requests to the API and dynamodb